core/storage/user/gae_models.py

```diff
@@ -925,6 +925,29 @@ def construct_model_id(cls, user_id, skill_id):
         """
         return '%s.%s' % (user_id, skill_id)
 
+    @classmethod
+    def export_data(cls, user_id):
+        """Exports the data from UserSkillMasteryModel
+        into dict format for Takeout.
+
+        Args:
+            user_id: str. The ID of the user whose data should be exported.
+
+        Returns:
+            dict. Dictionary of the data from UserSkillMasteryModel.
+        """
+
+        user_data = dict()
+        user_skills = cls.get_all().filter(cls.user_id == user_id).fetch()
+
+        for user_skill in user_skills:
+            user_data[user_skill.key.string_id()] = {
+                'skill_id': user_skill.skill_id,
+                'degree_of_mastery': user_skill.degree_of_mastery
+            }
+
```

Comment:
Actually I'm not sure about this format. I think it would be more intuitive to return a single dict which maps skill_id to degree_of_mastery. Can we do that?

Reply:
done